components/form.js

```diff
@@ -93,7 +93,7 @@ export function MarkdownInput ({ label, topLevel, groupClassName, onChange, setH
       const input = innerRef.current
       input.setSelectionRange(start, end)
     }
-  }, [innerRef?.current, selectionRange.start, selectionRange.end])
+  }, [selectionRange.start, selectionRange.end])
```

Comment:
Just fyi in modern versions of react all dependencies in useEffect must be specified.

Reply:
I think the move might be to not depend on innerRef.current and instead on just innerRef which shouldn't change on rerender

Reply:
> I think the move might be to not depend on innerRef.current and instead on just innerRef which shouldn't change on rerender

I agree. I tested it and it works: https://github.com/stackernews/stacker.news/pull/317/commits/b9286c59c084de7e8f136147b64cdc9706dc39b0